packages/ui/package.json

```diff
@@ -1,21 +1,33 @@
 {
   "name": "ui",
   "version": "0.0.0",
-  "main": "./index.tsx",
-  "types": "./index.tsx",
+  "sideEffects": [
+    "**/*.css"
+  ],
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": "./dist",
+    "./styles.css": "./dist/index.css"
+  },
   "license": "MIT",
   "scripts": {
-    "lint": "eslint *.ts*",
```

Comment:
why did we remove lint, clean and update?

Reply:
restored